notebooks/working/investigating_deconfounder.py

```diff
@@ -327,7 +298,7 @@
 display.display(sns.kdeplot(pvals[holdout_subjects]))
```

Comment:
Here, can you make the point horrendously obvious to viewers that the overall p-value is meaningless in this case.

Specifically, can you just plot the vertical line of the overall p-value on the plot with a label that says something along the lines of `"Observed: {:.2f}".format(pval_mode)`